locale/en/blog/announcements/interactive-2015-keynotes.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Keynotes for Node.js Interactive 2015 Announced
+date: 2015-11-20T09:00:00.000Z
+status: publish
+category: Annoucements
+slug: interactive-2015-programming
+layout: blog-post.hbs
+---
+
+> Keynotes from GoDaddy, IBM, NodeSource, Uber and More Featured At Inaugural Node.js Foundation Event December 8-9, 2015, in Portland, Ore.
+
+**SAN FRANCISCO, Nov. 20, 2015** – The [Node.js Foundation](https://nodejs.org/en/foundation/), a community-led and industry-backed consortium to advance the development of the Node.js platform, today announced the final keynotes and programming for [Node.js Interactive](http://events.linuxfoundation.org/events/node-interactive). The event will feature conversations and presentations on everything from the future of Node.js in IoT to collaborations between the community and the enterprise.
+
+Node.js is the runtime of choice for building mobile, web and cloud applications. The diversity of the technology and its capabilities are making it ubiquitous in almost every ecosystem from personal finance to robotics. To highlight changes with the platform and what’s to come, Node.js Interactive will focus on three tracks: Frontend, Backend and the Internet of Things (IoT). Highlights of these tracks available [here](https://nodejs.org/en/blog/announcements/interactive-2015-programming/); full track sessions [here](http://events.linuxfoundation.org/events/node-interactive/program/schedule).
+
+Node.js Interactive brings together a broad range of speakers to help experienced and novice Node.js users alike learn tips, best practices, new skills, as well as gain insight into future developments for the technology.  
+
+2015 Node.js Interactive keynotes include:
+
+### Day 1, December 8, 2015
+
+* Jason Gartner, Vice President, WebSphere Foundation and PureApplication Dev at IBM
+* James Snell, Open Technologies at IBM, “Enterprise Adoptions Rates and How They Benefit the Community”
+* Joe McCann, Co-Founder and CEO at NodeSource, “Convergence: Evolving Node.js with Open Governance and an Open Community”
+* Ashley Williams, Developer Community and Content Manager at npm
+* Tom Croucher, Engineer Manager at Uber, “Node.js at Uber”
+
+### Day 2, December 9, 2015
+
+* Mikeal Rogers, Node.js Foundation Community Manager at The Linux Foundation, “Node.js Foundation Growth and Goals”
+* Danese Cooper, Distinguished Member of Technical Staff - Open Source at PayPal and Node.js Foundation Chairperson
+* Panel Discussion with Node.js Foundation [Technical Steering Committee members](https://nodejs.org/en/foundation/tsc/)
```

Comment:
we don't have the most accurate count on that yet, but enough for a panel for sure :)